docs/server.js

```diff
@@ -21,8 +21,11 @@ if (development) {
   });
 
   app.use(function renderApp(req, res) {
+    res.header('Access-Control-Allow-Origin', target);
```

Comment:
I had commented out both lines `Access-*`, 
then  `npm run docs`,
then open in browser all pages with opened dev-console.
All is working and clear. Navigation, components rendering.

Then I had changed code on two pages: Components and Intro 
and those changes showed up in browser as it should be.

No errors at all.

What should I do to reproduce `cross origin request violations errors` ?